apps/nextjs-website/react-components/components/Header/Header.helpers.tsx

```diff
@@ -20,9 +22,14 @@ export const DialogBubble = ({
 }: DialogBubbleProps) => {
   const muiTheme = useTheme();
   const styles = useStyles(muiTheme);
+  
   return (
     <Stack
-      sx={{ ...styles.bubbleContainer, bgcolor: muiTheme.palette.common.white }}
+      sx={{ 
+        ...styles.bubbleContainer, 
+        bgcolor: muiTheme.palette.common.white, 
```

Comment:
```suggestion
        bgcolor: 'common.white', 
```
We can probably use theme-aware properties